packages/protocol/test-sol/unit/governance/network/Governance.t.sol

```diff
@@ -3383,6 +3517,59 @@ contract GovernanceTest_prepareHotfix_L2 is GovernanceTest {
 }
 
 contract GovernanceTest_resetHotfix is GovernanceTest {
+  bytes32 constant HOTFIX_HASH = bytes32(uint256(0x123456789));
+  bytes32 constant SALT = 0x657ed9d64e84fa3d1af43b3a307db22aba2d90a158015df1c588c02e24ca08f0;
+  bytes32 hotfixHash;
+  address validator1;
+  event HotfixRecordReset(bytes32 indexed hash);
+
+  function setUp() public {
+    super.setUp();
+
+    validator1 = actor("validator1");
+    governance.addValidator(validator1);
+    vm.prank(validator1);
+    accounts.createAccount();
+
+    vm.prank(accOwner);
+    governance.setSecurityCouncil(accCouncil);
+
+    hotfixHash = governance.getHotfixHash(
+      okProp.values,
+      okProp.destinations,
+      okProp.data,
+      okProp.dataLengths,
+      SALT
+    );
```

Comment:
Looks like there's some duplicated code between here and the L2 tests for this function. Abstract away into a `_setup` contract?